libs/printing/src/render.tsx

```diff
@@ -216,7 +216,7 @@ export async function renderToPdf(
     const headerHtml =
       headerTemplate &&
       (() => {
-        const { elementHtml, styleElement } =
```

Comment:
This looks like potentially a regression. I think we should use the `styleElement` from the rendering of the `headerTemplate` rather than using the variable of the same name from the outer scope.

Reply:
yeah I found this bug while testing something else just now too haha. I have a seperate branch with a fix and some added tests to pollbook to catch this in the future automatically. I'll pull the primary fix in and then keep the added tests and other changes on another branch to send out post-merge